deterministic-build.yml

```diff
@@ -0,0 +1,33 @@
+trigger:
+- master
+
+strategy:
+  matrix:
+    lin:
+      runtime: 'linux-x64'
+    mac:
+      runtime: 'osx-x64'
+    win:
+      runtime: 'win7-x64'
+
+pool:
+  vmImage: ubuntu-16.04
```

Comment:
Why was Ubuntu 16.04 chosen?

https://endoflife.software/operating-systems/linux/ubuntu - ubuntu 16.04 will reach its end-of-life in less than a year.

Reply:
As we are packaging on windows currently I changed the whole thing to win and to PowerShell script.